code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -44,8 +44,9 @@
 	icon_state = "random_bloodpack"
 
 /obj/item/reagent_containers/blood/random/Initialize()
+	icon_state = "bloodpack"
 	blood_type = pick("A+", "A-", "B+", "B-", "O+", "O-", "L")
-	. = ..()
+	return ..()
```

Comment:
that.... means the same thing.

Reply:
Saves a buffer I think, improving performance by one operation